sql_script/create_tables.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS user (
```

Comment:
table name starts with capital letter 
and column name is lower camel-case

Reply:
Done